Analysis/src/BaseEventAnalyzer.cpp

```diff
@@ -92,9 +96,19 @@ EventCategorySet BaseEventAnalyzer::DetermineEventCategories(EventInfo& event)
             }
         }
     }
+    auto vbf_counts = vbf_working_points;
+    if(is_VBF){
+        for(size_t vbf_index = 1; vbf_index <= 2; ++vbf_index) {
+            const auto& vbf_jet = event.GetVBFJet(vbf_index);
+            for(const auto& vbf_wp : vbf_working_points) {
+                if(bTagger->Pass(*vbf_jet, vbf_wp.first))
+                    ++vbf_counts[vbf_wp.first];
+            }
+        }
+    }
```

Comment:
should be integrated with lines 72-85. The result should be something like this:
```c++
bool is_VBF = false;
boost::optional<DiscriminatorWP> vbf_tag;
if(event.HasVBFjetPair()) {
    const auto vbf_jet_1 = event.GetVBFJet(1).GetMomentum();
    const auto vbf_jet_2 = event.GetVBFJet(2).GetMomentum();
    const auto deta_jj =  std::abs(vbf_jet_1.Eta() - vbf_jet_2.Eta());
    const auto m_jj = (vbf_jet_1 + vbf_jet_2).M();
    is_VBF = (m_jj > cuts::hh_bbtautau_Run2::VBF::mass_jj_loose && deta_jj > cuts::hh_bbtautau_Run2::VBF::deltaeta_jj);
    if(is_VBF) {
        const bool is_tight = m_jj > cuts::hh_bbtautau_Run2::VBF::mass_jj_tight && pass_vbf_trigger;
        vbf_tag = is_tight ? DiscriminatorWP::Tight : DiscriminatorWP::Loose;
    }
}
```